packages/@okta/vuepress-site/docs/reference/architecture-center/directory-coexistence/overview/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Directory Coexistence
+excerpt:
+   Minimize downtime while migrating user, group, and device profiles to Okta Universal Directory by keeping both source and target directories active during the process.
+---
+
+# Directory Coexistence
+
+Minimize downtime while migrating user, group, and device profiles to Okta Universal Directory by keeping both source and target directories active during the process.
+
+## Introduction
+
+Using [Universal Directory](https://www.okta.com/products/universal-directory/) as your company's sole identity provider enables benefits such as Single Sign-On, and centralized user management and authorization policies. However, migrating your users to Universal Directory takes time.
```

Comment:
I suggest keeping the user in developer.okta.com and send them to this reference in Concepts: https://developer.okta.com/docs/concepts/user-profiles/#what-is-the-okta-universal-directory

It provides more detail than our marketing page

Reply:
Fair - have changed the link